indexing-hadoop/src/test/java/io/druid/indexer/HadoopDruidIndexerConfigTest.java

```diff
@@ -245,7 +244,7 @@ public void testHashedBucketSelection()
     );
     final long timestamp = new DateTime("2010-01-01T01:00:01").getMillis();
     final Bucket expectedBucket = config.getBucket(new MapBasedInputRow(timestamp, dims, values)).get();
-    final long nextBucketTimestamp = QueryGranularities.MINUTE.next(QueryGranularities.MINUTE.truncate(timestamp));
+    final long nextBucketTimestamp = Granularity.MINUTE.increment(Granularity.MINUTE.truncate(new DateTime(timestamp))).getMillis();
```

Comment:
i didn't think this was widely used in such a manner that warranted this method. also abstracting this means modifying code around the call sites. also, right now all the methods interface with DateTime, i didn't want to have a one off that took in timestamp. if you think this is a must have, then i'll look into it.